lib/plugins/platform/platform.js

```diff
@@ -65,14 +65,36 @@ class Platform {
     if (this.provider) {
       this.hooks = {
         'after:deploy:deploy': this.publishService.bind(this),
-        'after:remove:remove': this.removeService.bind(this),
+        'after:remove:remove': this.archiveService.bind(this),
       };
     }
   }
 
-  removeService() {
-    // TODO implement platform removal here
-    return BbPromise.resolve();
+  archiveService() {
+    const authToken = this.getAuthToken();
+    const publishFlag = selectServicePublish(this.serverless.service);
+    if (!authToken || !publishFlag) {
+      // NOTE removeService is an opt-in feature and no warning is needed
```

Comment:
Is this `removeService` or `archiveService`? Asking because I'm wondering if we implement a removal in the Framework as well.

Reply:
This is actually `archiveService`, as far as I'm aware `sls remove` should archive the service, which is what it does.

Reply:
Exactly, the only way to really remove a service from the Platform is to do it in the UI.

Reply:
Gotcha! Then we might want to update this comment here to avoid confusion when someone else stumbled upon it.